mapper.html

```diff
@@ -0,0 +1,204 @@
+<!--
+Copyright 2022 Google LLC.
+Copyright (c) Microsoft Corporation.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<div class="card">
+  <div class="item">
+    <div class="text_1">
+      BiDi mapper is controlling this tab
+      <div class="right">
+        <a
+          target="_blank"
+          title="BiDi-CDP Mapper GitHub Repository"
+          href="https://github.com/GoogleChromeLabs/chromium-bidi"
+          >GoogleChromeLabs / chromium-bidi</a
+        >
+      </div>
+    </div>
+    <div class="text_2">
+      Closing or reloading it will stop the BiDi process.
+    </div>
+  </div>
+  <div class="divider"></div>
+  <div class="item">
+    <div class="text_3">
+      Debug information
+      <div class="right">
+        <a
+          target="_blank"
+          title="Show/hide debug info"
+          href="#"
+          onclick="document.getElementById('debug').style.display=(document.getElementById('debug').style.display==='none'?'block':'none'); return false;"
+          >show / hide debug info</a
+        >
```

Comment:
Could we just use `<details>`/`<summary>`?

Reply:
Switched to `details/summary`, thanks!